Prog/Equity_Compile_ACS_for_profile.sas

```diff
@@ -923,6 +967,39 @@
     %Moe_prop_a( var=PctOwnerOccupiedHU&race._m_&_years., mult=100, num=NumOwnerOccupiedHU&race._&_years., den=NumOccupiedHsgUnits&race._&_years., 
                        num_moe=mNumOwnerOccupiedHU&race._&_years., den_moe=mNumOccupiedHsgUnits&race._&_years., label_moe =Homeownership rate &rname.(%) MOE &y_lbl.);
     
+			%if &lastyear. >= 2023 %then %do; 
+
+		%Pct_calc( var=PctMortcstbrden&race., label=% Owners with Mortgage who are cost-burdened &rname., num=Mortcstbrden&race., den=mortcstbrdencalc&race., years=&_years. )
+			%Moe_prop_a( var=PctMortcstbrden&race._m_&_years., mult=100, num=Mortcstbrden&race._&_years., den=mortcstbrdencalc&race._&_years., 
+	                       num_moe=mMortcstbrden&race._&_years., den_moe=mMortcstbrdencalc&race._&_years., label_moe =% Owners with Mortgage who are cost-burdened &rname. MOE &y_lbl.);
+		%Pct_calc( var=PctMortSCstbrden&race., label=% Owners with Mortgage who are severely cost-burdened &rname., num=Mortsvrecstbrden&race., den=mortcstbrdencalc&race., years=&_years. )
+			%Moe_prop_a( var=PctMortSCstbrden&race._m_&_years., mult=100, num=Mortsvrecstbrden&race._&_years., den=mortcstbrdencalc&race._&_years., 
+	                       num_moe=mMortSvrecstbrden&race._&_years., den_moe=mMortcstbrdencalc&race._&_years., label_moe =% Owners with Mortgage who are severely cost-burdened &rname.  MOE &y_lbl.);
+
+		%Pct_calc( var=PctNomortcstbrden&race., label=% Owners without Mortgage who are cost-burdened &rname., num=NoMortcstbrden&race., den=mortcstbrdencalc&race., years=&_years. )
+			%Moe_prop_a( var=PctNomortcstbrden&race._m_&_years., mult=100, num=Nomortcstbrden&race._&_years., den=mortcstbrdencalc&race._&_years., 
```

Comment:
Same as above. Use `nomortcstbrdencalc&race` and `mnomortcstbrdencalc&race` throughout.